src/main/java/com/rackspace/ceres/app/web/MetadataController.java

```diff
@@ -33,31 +39,53 @@
 
   private final MetadataService metadataService;
 
+  private final Environment environment;
+
+  private final AppProperties appProperties;
```

Comment:
May not need to autowire `appProperties` since the `#{appProperties ...` expression should actually be accessing the beans in the app context.

Reply:
Ohh Yeahhh...That's right